src/HackF5.UnitySpy/Detail/TypeDefinition.cs

```diff
@@ -65,6 +65,7 @@ public TypeDefinition([NotNull] AssemblyImage image, IntPtr address)
             // Get the generic type arguments
             if (this.TypeInfo.TypeCode == TypeCode.GENERICINST)
             {
+                this.fieldCount = this.ReadInt32(96);
```

Comment:
Here I can't simply use 96 for the `TypeDefinitionFieldCount` offset since this line isn't always executed at the same time as the line that uses `TypeDefinitionFieldCount`